asset_dashboard/static/js/components/maps/update.js

```diff
@@ -0,0 +1,132 @@
+import ReactDOM from 'react-dom'
```

Comment:
Can we call this file a noun, such as `clippableMap.js`?

Reply:
I renamed it `AssetsMap.js` to match the exported component name. I renamed the component to `AssetsMap` since "update" doesn't completely describe that due to the searched assets showing up on the map, too. I'm starting to name all the files after the exported names, [following the AirBnb style guide](https://github.com/airbnb/javascript#naming--filename-matches-export). And changing the unpythonic snake_case filenames to the ES6 style of camelCase for javascript files that aren't components.